content/blog/2022-09-29-contribute-to-open-source-with-localstack-hacktoberfest/index.md

```diff
@@ -0,0 +1,60 @@
+
+We are now looking for more contributions from the community, whether it is a minor documentation fix, bug triaging to feature improvements. We appreciate all contributions coming in, and we are looking to support all new-time contributors who are looking to contribute through Hacktoberfest.
+
+## Getting started with Hacktoberfest
+
+LocalStack is participating in Hacktoberfest with the following projects: LocalStack, LocalStack Docs, LocalStack Samples & Terraform Samples. We are not just accepting code-based contributions but also no-code contributions like documentation, design, bug triage, and more! Contributors with four or more merged pull requests will be considered for the Hacktoberfest prizes if they follow all Hacktoberfest guidelines.
+
+Follow these simple steps to get started:
+
+1.  Register on the  [Hacktoberfest website](https://hacktoberfest.com/) between September 26 and October 31, 2022.
+2.  Look through our above projects and create/find issues you would like to work on!
+3.  Contribute to the projects by following our [good first issue tag]() between October 1 to October 31. 
+4.  Ensure follow-ups with the maintainers on respective pull requests to ensure they are merged before the end of Hacktoberfest.
+
+After your pull request is merged, share your achievement on the `#community` channel of our [community slack](https://slack.localstack.cloud). After the end of Hacktoberfest, all the contributors will be acknowledged in a public community post over our [Discussion pages](https://discuss.localstack.cloud).
+
+But what's new apart from the official Hacktoberfest prizes? Your contributions call for more awesome schwag! To celebrate your contributions during Hacktoberfest, we have some limited edition swag up for grabs!
+
+- **4 contributions**: LocalStack T-Shirt 
+- **10 contributions**: LocalStack T-Shirt + Amazon $25 gift-card
+- **Best contributions**: LocalStack Gift Box + Amazon $50 gift-card
+
+While selecting the best contribution is tricky, we recommend you to look apart from regular LocalStack issues to find something significant to work on. Ideas can vary from a new AWS service (you can scaffold it quickly with ASF), a significant bug fix, new examples, blogs, or more! A contribution would be considered irrespective of whether we merged a pull request or not.
+
+## How should we ask for help? 
+
+To help our ever-growing community of users and developers, we have made significant new additions to our [developer guide](https://docs.localstack.cloud/developer-guide/) to make it easy for you to contribute to LocalStack! If you are looking to set up LocalStack for the first time, here is an increasingly helpful walkthrough by Thomas guiding you to kick-start your LocalStack development with an IDE.
```

Comment:
I guess we're missing a link to the getting-started video here..?

Reply:
Added now!